tock/tock/templates/utilization/unit_utilization_cell.html

```diff
@@ -0,0 +1,6 @@
+<td>
```

Comment:
Super minor, but for tidiness, maybe we should put these includes in a `utlization/includes/` folder?